build/ansible/pmm2/post-build-actions.yml

```diff
@@ -44,6 +44,28 @@
     - name: Enable repo                | Enable release repo for pmm2-client
       command: percona-release enable {{ pmm_client_repos_final }}
 
+    - name: Detect Docker
+      set_fact:
+        is_docker: '{{ lookup("file", "/srv/pmm-distribution") == "docker" }}'
+      no_log: yes
+
+    - name: Install glibc-langpack-en  | EL9
+      dnf:
+        name: glibc-langpack-en
+        state: present
+        update_cache: yes
+      when:
+        - not is_docker
+        - ansible_distribution == 'OracleLinux' or ansible_distribution == 'AlmaLinux'
```

Comment:
Do we still need AlmaLinux mentions?

Reply:
I believe yes, until the other PR, that migrates our AMI to OL9, is merged. Then we can remove all mentions of AlamaLinux.

Reply:
> Can we merge #2306 then?

Sure, once it's tested :)